actions/npm-version-bump/test/main.test.js

```diff
@@ -0,0 +1,158 @@
+      const initialVersion = "1.3.0"
+
+      const packageJson = {
+        name: "@keep-network/keep-core",
+        version: initialVersion,
+      }
+
+      writeFileSync(testFilePath, JSON.stringify(packageJson))
+
+      const newVersion = await execute(
+        dirname(testFilePath),
+        isPrerelease,
+        defaultEnvironment,
+        branch,
+        commit
+      )
+
+      expect(newVersion).to.not.equal(initialVersion)
+
+      const resultFile = require(testFilePath)
+
+      expect(resultFile.version).to.equal(newVersion)
+    })
+
+    it("updates version for published base version", async () => {
+      await verify("1.7.0", "1.7.1-pre.0+feature-branch-2.1234abcd7890XYZ")
+    })
+
+    it("updates version for not-published base version", async () => {
+      await verify("1.7.1", "1.7.2-pre.0+feature-branch-2.1234abcd7890XYZ")
+    })
+
+    it("updates version for published pre version", async () => {
+      await verify(
+        "1.7.0-pre.7",
+        "1.7.1-pre.0+feature-branch-2.1234abcd7890XYZ"
+      )
+    })
+
+    it("updates version for published pre version that has newer rc version", async () => {
+      await verify("1.3.0-pre", "1.3.1-pre.0+feature-branch-2.1234abcd7890XYZ")
+    })
+
+    it("updates version for rc version", async () => {
+      await verify("1.7.1-rc.0", "1.7.1-pre.0+feature-branch-2.1234abcd7890XYZ")
+    })
+
+    it("bumps build version for already published pre version", async () => {
+      await verify("1.8.0-pre", "1.8.0-pre.17+feature-branch-2.1234abcd7890XYZ")
+    })
+
+    it("updates version for not published version with lesser preid", async () => {
+      await verify("9.0.0-dev", "9.0.0-pre.0+feature-branch-2.1234abcd7890XYZ")
+    })
+
+    it("updates version for not published version with the same preid", async () => {
+      await verify("9.0.0-pre", "9.0.0-pre.0+feature-branch-2.1234abcd7890XYZ")
+    })
+
+    it("updates version for not published version with greater preid", async () => {
+      await verify("9.0.0-rc", "9.0.0-pre.0+feature-branch-2.1234abcd7890XYZ")
+    })
+
+    it("updates version for not published version with base only", async () => {
+      await verify("9.0.0", "9.0.1-pre.0+feature-branch-2.1234abcd7890XYZ")
+    })
+
+    it("updates version for non-published preid", async () => {
+      await verify(
+        "1.3.1-pre",
+        "1.3.1-testenv.0+feature-branch-2.1234abcd7890XYZ",
+        "testenv"
+      )
+    })
+
+    it("updates version for non-published preid and environment", async () => {
+      await verify(
+        "1.3.1-test",
+        "1.3.1-testenv.0+feature-branch-2.1234abcd7890XYZ",
+        "testenv"
+      )
+    })
+
+    it("updates version for already published environment that don't match initial preid", async () => {
+      await verify(
+        "1.1.2-pre",
+        "1.1.2-ropsten.1+feature-branch-2.1234abcd7890XYZ",
+        "ropsten",
+        "@keep-network/tbtc"
+      )
+    })
```

Comment:
We're getting failure of this test. I think it is related to the fact, that in the [tbtc's npm registry](https://www.npmjs.com/package/@keep-network/tbtc) we now have the `1.1.2-ropsten.2` package published, and the expected value is no longer a valid one.

We could update the test by changing the expected value:
```suggestion
    it("updates version for already published environment that don't match initial preid", async () => {
      await verify(
        "1.1.2-pre",
        "1.1.2-ropsten.3+feature-branch-2.1234abcd7890XYZ",
        "ropsten",
        "@keep-network/tbtc"
      )
    })
```
But this would mean that we need to update the test again when we publish next `1.1.2` package with `ropsten` preid (which is quite likely) Maybe it would be better to change the test to this? @nkuba, what do you think?

```suggestion
    it("updates version for already published environment that don't match initial preid", async () => {
      await verify(
        "0.15.0-pre",
        "0.15.0-rc.1+feature-branch-2.1234abcd7890XYZ",
        "rc",
        "@keep-network/tbtc"
      )
    })
```

Reply:
Yes, let's use latter.

Reply:
Change applied in 3987a934e3cd770e853dd412ff622b77908bd188. Previously failing test is now passing.